src/bitcoin-cli.cpp

```diff
@@ -65,6 +65,7 @@ static bool AppInitRPC(int argc, char* argv[])
     ParseParameters(argc, argv);
     if (argc<2 || mapArgs.count("-?") || mapArgs.count("-h") || mapArgs.count("-help") || mapArgs.count("-version")) {
         std::string strUsage = _("Zcash RPC client version") + " " + FormatFullVersion() + "\n";
+        strUsage += "\n" + _("In order to ensure you are adequately protecting your privacy when using Zcash, please see https://z.cash/support/security/index.html") + "\n";
```

Comment:
`strprintf(_("..., please see %s"), "https://z.cash/support/security/index.html")` so the URL isn't part of the translation string.